m4/Cargo.lock

```diff
@@ -0,0 +1,618 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+version = 3
+
```

Comment:
Something is wrong related to workspaces, if this file exists.

The only Cargo.lock should be the workspace Cargo.lock at the top-level of the tree.